server/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
```

Comment:
Docker Compose is a tool for defining and managing multi-container Docker applications. It uses a YAML file (usually named docker-compose.yml) to configure your application's services, and then provides a single command to start up all the services from your configuration.
The docker compose won't be used in the real deployment, it's here only for local development purposes. Shall I document it in the readme file for the server module?

Reply:
Sorry for ambiguous wording of the questions. 😄 
What I meant by it, that it would be nice to have here a short description of the purpose/goal of the created docker images.
And yeah, a brief mention in the `README.md` about the existence of the file would not hurt.

As a side note - location of the file? (but it can be moved whenever later)

Reply:
Added a section about observability to the readme file for server module.